stage1/dbus/org.freedesktop.systemd1.service

```diff
@@ -0,0 +1,11 @@
+#  This file is part of systemd.
```

Comment:
What is this file for? It looks like it preventing systemd dbus manager to run. Why?